src/travis.py

```diff
@@ -31,7 +31,7 @@ def _get_travis_token() -> str:
     )
     token: str = kms_client.decrypt(
         name=resource_name, ciphertext=b64decode(os.environ[env_var])
-    ).plaintext.decode()
+    ).plaintext.decode().strip()
```

Comment:
The decrypted key has `\n` in it. I'm not sure how I avoided that originally - removing the `\n` in `--plaintext-file=token.txt` had no impact. Don't think it matters much, though.